modules/platforms/build.gradle

```diff
@@ -267,3 +276,71 @@ private void updatePythonVersion(String version) {
     def versionFile = file("$projectDir/python/pyignite_dbapi/_version.txt")
     versionFile.text = simpleProjectVersion(version)
 }
+
+def buildWindowsDbApiWheels = tasks.register('buildWindowsDbApiWheels', Exec) {
```

Comment:
There are Linux and Windows tasks. Does it mean that to prepare a release, we have to run both, using two different OS?

Reply:
Yes, for now. I will try and add a docker for Windows wheels in future, but this is enough for automating the release on TC.

Reply:
The problem is, we don't use TC to prepare Ignite release. We run the following command locally to build and sign artifacts, using PMC own signature:
```
./gradlew -PprepareRelease prepareRelease
```

Ideally, Python should be a part of this.

Reply:
I know. It requires some additional work, including work with TC agents. There is a ticket for that: https://issues.apache.org/jira/browse/IGNITE-24542